index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
```

Comment:
- Fix your Linter issues in your pull request.
- Provide a Pull Request description about your features implemented in your project.